yb-voyager/src/query/queryissue/detectors_ddl.go

```diff
@@ -773,7 +789,7 @@ func (d *IndexIssueDetector) DetectIssues(obj queryparser.DDLObject) ([]QueryIss
 					}
 				}
 				//TODO: separate out the Types check of Hotspot problem and the Range sharding recommendation
-				if tableHasHotspotTypes && idx == 0 {
+				if tableHasHotspotTypes && idx == 0 && !tm.IsPartitioned() {
```

Comment:
add a comment here explaining why we're skipping partitioned tables. 
pls do so for other similar instances as well.